CONTRIBUTING.md

```diff
@@ -50,7 +59,16 @@ Milestones are used to manage sprints, which are two weeks long. Sprint mileston
 - **13:** Issues that are more complicated and need some workflow or design planning. These issues usually need additional unit tests written.
 - **40:** If an issue is this complicated it should be converted into an epic.
 
-### Getting a development environment set up
+## Code base
+
+Our code base is written entirely in TypeScript and we should aim to have it that way. We have configured the project to help adhere to some conventions and formatting. Here are additional items to follow:
```

Comment:
> Our code base is written entirely in TypeScript and we should strive to keep it that way in order to benefit from type protection. We have configured the project to help adhere to some conventions and formatting. Here are additional items to follow:

?

Reply:
All of the stuff in these 3 comments were taken directly from the project guidelines, e.g. https://github.com/Esri/calcite-components/blob/master/PROJECT_GUIDELINES.md#code-base

This PR is just about removing repeat info without loosing any content. Next I will make a PR to fix wording. Is that okay?

Reply:
I'll make sure to fix the three things you mentioned in the next PR

Reply:
Yeah thats fine 👍